### PR TITLE
[API][SIMS #1312] Remove delete cascade

### PR DESCRIPTION
**Pull request by henryoforeh-aot**
- Created migration script to remove all existing delete cascade rules.
- Created script to roll back by adding delete cascade rules.
- Tested delete cascade between application and assessment after removing delete cascade rules.

### CODE REVIEW

File: sources/packages/api/src-sql/DeleteCascades/Add-cascade-rules.sql [hunk 1]
```diff
@@ -0,0 +1,203 @@
+-- Create reference between application_exception_requests and application_exceptions --
```
Comment (andrewsignori-aot) [inline sources/packages/api/src-sql/DeleteCascades/Add-cascade-rules.sql:1]: When the rollback script looks like something that we are adding we usually add some prefixes like `Rollback-drop-cascade-rules.sql`.

Comment (henryoforeh-aot) [reply sources/packages/api/src-sql/DeleteCascades/Add-cascade-rules.sql:1]: I will update

Comment (ann-aot) [inline sources/packages/api/src-sql/DeleteCascades/Add-cascade-rules.sql]: periods for all comments

**Comment by henryoforeh-aot** [comment]
Creating a new PR

**Pull request closed without merging**